lib/events/message.js

```diff
@@ -16,6 +20,21 @@ module.exports = (data, cb) => {
   // }).catch(function (e) {
   //   cb(e)
   // })
+  if (data.event.text.includes('roll')) {
```

Comment:
I'm leaning towards the DWIM one cause it means no need for further action for users. If there's disagreement about it, we can go for the former with a pretty trivial PR.